app/controllers/lists_controller.rb

```diff
@@ -0,0 +1,59 @@
+class ListsController < ApplicationController
+  before_action :authenticate_user!
+
+  def show
+    @lists = current_user.lists
```

Comment:
Line 5 looks like it'd belong in an `index` action as its multiple `List`s, not a single list.